packages/ui/src/components/va-toast/plugin/index.ts

```diff
@@ -1,21 +1,29 @@
 import { App } from 'vue'
 import { createToastInstance, closeById, closeAllNotifications, NotificationOptions } from '../toast'
 
-export const VaToastPlugin = {
-  install (app: App) {
-    app.config.globalProperties.$vaToast = {
-      /** Returns toast instance id */
-      init (options: string | NotificationOptions) {
-        return createToastInstance(options, app?._context)
-      },
+const createVaToastContext = (app: App) => ({
+  /** Returns toast instance id */
+  init (options: string | NotificationOptions) {
+    return createToastInstance(options, app?._context)
+  },
+
+  close (id: string) {
+    closeById(id)
+  },
 
-      close (id: string) {
-        closeById(id)
-      },
+  closeAll (allApps = false) {
+    closeAllNotifications(allApps ? undefined : app?._context)
+  },
```

Comment:
closeAllNotifications doesn't seem to accept any arguments. What the point of this expression?

Reply:
My bad. Took this function from [useToast hook PR#1464](https://github.com/epicmaxco/vuestic-ui/pull/1464/files#diff-d6cb9ad66be91ffcf23cbdbc6e22f8f1449d8ad2b871c8b811973ea6b9b94f98R126)

Please, take a look on idea, but not on implementation. This is important architecture rework, we want it to be flexible as maximum.